src/coffee/client/modules/todo/todo_list_module.coffee

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2015 by Looker Data Services, Inc.
+# All rights reserved.
+#
+
+angular   = require 'angular'
+templates = require '../../templates'
+{EVENT}   = require '../../../constants'
+_         = require '../../../underscore'
+
+############################################################################################################
+
+### ex: TodoList flow
+1. TodoListController listens to TodoListStore and updates its `$scope.todos` when anything changes
+2. TodoListController listens to TodoItemStore fires TodoList's loadAll action when todos are added/removed
+3. Controller fires removeItem/doItem actions based on user events
+
+############################################################################################################
+
+todo = angular.module('todo')
+
+todo.factory 'TodoListStore', (Todo, TodoItemStore, TodoListActions, ReadOnlyView, reflux) ->
+    reflux.createStore
+
+        listenables: TodoListActions
+
+        onLoadAll: ->
+            @_todos = TodoItemStore._items
+            @trigger(EVENT.CHANGE)
+
+
+todo.factory 'TodoListActions', (reflux) ->
+    reflux.createActions ['loadAll']
+
+
+todo.directive 'todoList', () ->
+    restrict: 'E'
+    scope: {}
+    template: templates['todo-list']
+    controllerAs: 'controller'
+    controller: ($scope, TodoListStore, TodoListActions, TodoItemStore, TodoItemActions) ->
+        TodoItemStore.$listen($scope, (event, id) ->
+            TodoListActions.loadAll() if event == 'add' || event == 'remove' # (2)
+        )
+
+        TodoListStore.$listen($scope, (event, id) ->
+            $scope.todos = TodoListStore._todos  # (1)
+        )
+
+        $scope.removeTodoItem = (id) -> # (3)
```

Comment:
I think it would be better to have each to-do list item be it's own directive.  Each could accept a scope-bound attribute for its `id`, and its controller could then add a `remove` and `makeDone` method which can just look on the scope for the correct `id`.